mulighetsrommet-api/src/main/resources/db/migration/V230__to_trinnskontroll.sql

```diff
@@ -0,0 +1,32 @@
+drop view if exists tilsagn_admin_dto_view;
+drop view if exists tilsagn_arrangorflate_view;
+
+create type besluttelse as enum ('GODKJENT', 'AVVIST');
+create type to_trinnskontroll_type as enum (
+    'OPPRETT_TILSAGN',
+    'ANNULLER_TILSAGN',
+    'OPPRETT_UTBETALING'
+);
+
+create table to_trinnskontroll
```

Comment:
Jeg sliter litt med lesbarheten "to trinnskontroll". Lurer på om vi kunne kalt det for noe annet. F.eks. noe i baner av "tiltaksøkonomi-logg" eller lignende